tests/testutil/require.go

```diff
@@ -161,15 +207,75 @@ func RequireDatabaseToFunction(t *testing.T, crdbCluster CockroachCluster, dbNam
 // RequireCRDBToFunction creates a database, a table and insert two rows if it is a fresh install of the cluster.
 // If certificate is rotated and cluster rolling restart has happened, this will check that existing two rows are present.
 func RequireCRDBToFunction(t *testing.T, crdbCluster CockroachCluster, rotate bool) {
-	db := getDBConn(t, crdbCluster, "system")
+	db := getDBConn(t, crdbCluster, "system", "")
 
 	if rotate {
 		t.Log("Verifying the existing data in the database after certificate rotation")
 	}
 
 	// Create database only if we are testing crdb install
 	if !rotate {
-		if _, err := db.Exec("CREATE DATABASE test_db"); err != nil {
+		if _, err := db.Exec("CREATE DATABASE IF NOT EXISTS testdb"); err != nil {
```

Comment:
```suggestion
		if _, err := db.Exec("CREATE DATABASE IF NOT EXISTS test_db"); err != nil {
```
Looks like this is causing the test to fail on CI.

Reply:
Thanks, fixed!